packages/mongoose-store/src/query-builder.ts

```diff
@@ -0,0 +1,85 @@
+// TODO: finish adding types to this
```

Comment:
I do not think that we should have it. This query builder was designed to support fh.db like api.
This api is really old and limited and we should no longer use it in new Raintacher. It was added only to support backwards compatibility and let it work with fh-sync.